### ISSUE TITLE
Replace cStringIO with BytesIO in twisted/spread

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8617]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8617|
|Type|enhancement|
|Created|2016-07-18 06:15:55Z|
|Branch|https://github.com/twisted/twisted/tree/8617-rodrigc-spread|

In twisted/spread, cStringIO is used.  This is gone on Python 3.
BytesIO should be used instead.

<details><summary>Searchable metadata</summary>

```
trac-id__8617 8617
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8617_rodrigc_spread 8617-rodrigc-spread
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468822555353696 1468822555353696
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/346

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 229a9f522beaa19c6b288eaa0fc13294fa8606c0
```
#!CommitTicketReference repository="" revision="229a9f522beaa19c6b288eaa0fc13294fa8606c0"
Merge pull request #346 from twisted/8617-rodrigc-spread

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8617

Replace cStringIO with BytesIO
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
